src/OrbitVulkanLayer/VulkanWrapper.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_VULKAN_LAYER_VULKAN_WRAPPER_H_
+#define ORBIT_VULKAN_LAYER_VULKAN_WRAPPER_H_
+
+#include "vulkan/vulkan.h"
+
+namespace orbit_vulkan_layer {
+
+// This class provides a wrapper for calls directly into the Vulkan loader.
+// It is used, such that we can fake the called Vulkan functions in the tests.
+
```

Comment:
```suggestion
//
```
(So that it's clear it's part of the same comment.)